docs/development/extensions-core/stats.md

```diff
@@ -181,3 +206,14 @@ To acquire standard deviation from variance, user can use "stddev" post aggregat
   ]
 }
 ```
+
+#### Druid SQL
+
+```SQL
+SELECT
+  alias,
+  VARIANCE("index") AS index_var
+FROM "testing"
+WHERE TIME_IN_INTERVAL(__time, '2016-03-06T00:00:00/2016-03-06T23:59:59')
```

Comment:
End date is exclusive, so I would think this should be start of the next day? Same for the native query:
```suggestion
WHERE TIME_IN_INTERVAL(__time, '2016-03-06/2016-03-07')
```